app/views/user_rules/edit.html.erb

```diff
@@ -12,24 +11,26 @@
     <li><%= link_to "Promote User", new_workflow_user_rule_user_rule_effect_path(@workflow, @user_rule, action_type: 'promote_user') %></li>
   </ul>
 </div>
-<table class="table table-hover table table-sm">
-    <thead class="thead-inverse">
-      <tr>
-        <th>Effect</th>
-        <th>Actions</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @user_rule.user_rule_effects.each do |rule_effect| %>
-      <tr>
-        <td><%= render partial: 'effect', locals: {effect: rule_effect.effect} %></td>
-        <td>
-          <%= link_to "Edit", [:edit, @workflow, @user_rule, rule_effect], class: 'btn btn-primary' %>
-          <%= link_to "Destroy", [@workflow, @user_rule, rule_effect], method: :delete, data: {confirm: "Are you sure?"}, class: 'btn btn-danger' %>
-        </td>
-      </tr>
-      <% end %>
+<table class="table table-striped table-sm">
+  <colgroup>
+    <col />
+    <col style="width: 4em; " />
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Effect</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @user_rule.user_rule_effects.each do |rule_effect| %>
+    <tr>
+      <td><%= render partial: 'effect', locals: {effect: rule_effect.effect} %></td>
+      <td>
+        <%= link_to "", [:edit, @workflow, @user_rule, rule_effect], class: 'glyphicon glyphicon-cog text-warning' %>
```

Comment:
How accessible is this?

Reply:
I'll try to figure out how this acts in a screen reader